src/day-1-toy/modules.py

```diff
@@ -9,10 +9,10 @@
 
```

Comment:
Don't forget this `sys.argv` prompt here.

Reply:
Okay. I'll go back and do that one when I'm finishing up the first day of that.